ajax/libs/jReject/package.json

```diff
@@ -35,5 +35,5 @@
       "images/browser*.gif"
     ]
   },
-  "license": "MIT", "GPL-2.0"
+  "license": {"MIT", "GPL-2.0"}
```

Comment:
See https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/vis/package.json#L32-L35 for the right format.